config/environments/test.rb

```diff
@@ -41,4 +41,7 @@
 
   # Raises error for missing translations
   config.action_view.raise_on_missing_translations = true
+
+  # Enables secret share feature flag
+  config.secret_share_enabled = true
```

Comment:
I think it would be worth mentioning in our documentation that we have this `secret_share_enabled` feature flag!

Reply:
Hopefully the flag will be short-lived... it can be deleted once the feature is production ready.